pygmt/clib/session.py

```diff
@@ -314,6 +314,22 @@ def get_libgmt_func(self, name, argtypes=None, restype=None):
             function.restype = restype
         return function
 
+    def set_allocmode(self, family, obj):
```

Comment:
Please add these methods to `doc/api/index.rst` so that they will appear on https://www.pygmt.org/latest/api/index.html#gmt-c-api:

- [ ] set_allocmode
- [ ] read_virtualfile
- [ ] virtualfile_from_gmtgrid
- [ ] virtualfile_to_gmtgrid
- [ ] virtualfile_from_xrgrid

Reply:
Done in f3fb38d0ffe691ee8f810490bfc531d850c1ba24.